.env.dist

```diff
@@ -1,6 +1,7 @@
 GITHUB_OAUTH_TOKEN=token_to_be_changed
 DEK_KIT_TOKEN=token_to_be_changed
 SLACK_HOOK_URL=hook_to_be_changed
+SLACK_TOKEN=token_to_be_changed_get_it_at_https://api.slack.com/apps/A018X725S0K/install-on-team
```

Comment:
See https://github.com/sonata-project/dev-kit/pull/812#issuecomment-674378059

I think, the `get_it_at` part should be in a `How to install locally` part of the Readme.